app/src/organisms/Devices/PipetteCard/index.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { LEFT } from '../../../redux/pipettes'
+import {
+  Box,
+  Flex,
+  DIRECTION_ROW,
+  ALIGN_START,
+  DIRECTION_COLUMN,
+  TEXT_TRANSFORM_UPPERCASE,
+  SPACING,
+  FONT_WEIGHT_REGULAR,
+  FONT_SIZE_CAPTION,
+  TYPOGRAPHY,
+  COLORS,
+  useOnClickOutside,
+  InstrumentDiagram,
+} from '@opentrons/components'
+import { OverflowBtn } from '../../../atoms/MenuList/OverflowBtn'
+import { StyledText } from '../../../atoms/text'
+import { PipetteOverflowMenu } from './PipetteOverflowMenu'
+
+import type { Mount } from '../../../redux/pipettes/types'
+import type { PipetteModelSpecs } from '@opentrons/shared-data'
+
+interface PipetteCardProps {
+  pipetteInfo: PipetteModelSpecs | null
+  mount: Mount
+  robotName: string
+}
+
+export const PipetteCard = (props: PipetteCardProps): JSX.Element => {
+  const { t } = useTranslation('device_details')
+  const [showOverflowMenu, setShowOverflowMenu] = React.useState(false)
+  const { pipetteInfo, mount, robotName } = props
+  const pipetteName = pipetteInfo?.displayName
+  const pipetteOverflowWrapperRef = useOnClickOutside({
+    onClickOutside: () => setShowOverflowMenu(false),
+  }) as React.RefObject<HTMLDivElement>
+
+  return (
+    <Flex
+      backgroundColor={COLORS.background}
+      borderRadius={SPACING.spacing2}
+      marginBottom={SPACING.spacing3}
+      marginLeft={SPACING.spacing2}
+      marginRight={SPACING.spacing2}
+      width={'100%'}
+      data-testid={`PipetteCard_${pipetteName}`}
+    >
+      <Box
+        padding={`${SPACING.spacing4} ${SPACING.spacing3} ${SPACING.spacing4} ${SPACING.spacing3}`}
```

Comment:
```suggestion
        padding={`${SPACING.spacing4} ${SPACING.spacing3}`}
```
This is equivalent  what's written using padding's two value shorthand